tools/gsc-high_dimensions_visualisation/high_dim_visu.xml

```diff
@@ -54,19 +78,62 @@
+					<option value="FALSE">False</option>
+				</param>
+				<param name="Rtsne_pca_center"  type="select" label="pca centering (t-SNE)" help="Should data be centered before pca is applied? " > 
+					<option value="TRUE" selected="true">Yes</option>
+					<option value="FALSE">False</option>
+				</param>
+				<param name="Rtsne_pca_scale"  type="select" label="pca scaling (t-SNE)" help="Should data be scaled before pca is applied? " > 
+					<option value="YES">Yes</option>
+					<option value="FALSE" selected="true">False</option>
+				</param>
+				<param name="Rtsne_normalize"  type="select" label="pca centering (t-SNE)" help="Should data be normalized internally prior to distance calculations? " > 
+					<option value="TRUE" selected="true">Yes</option>
+					<option value="FALSE">False</option>
+				</param>
+				<param name="Rtsne_exaggeration_factor" value="12.0" type="float" label="Exageration factor" help="Exaggeration factor used to multiply the P matrix in the first part of the optimization" />
+				
             </when>
             <when value="HCPC">
-                <param name="npc" value="5" type="integer" label="Number of principal components to keep"
+                <param name="HCPC_npc" value="5" type="integer" label="Number of principal components to keep"
                        help="The number of dimensions which are kept for HCPC analysis (default=5)" />
-                <param name="ncluster" value="-1" type="integer" label="Number of clusters in Hierar. Clustering"
+                <param name="HCPC_ncluster" value="-1" type="integer" label="Number of clusters in Hierar. Clustering"
                        help="nb.clust, the number of clusters to consider in the hierarchical clustering. (default : -1, let HCPC to optimize the number)" />
+				<param name="HCPC_metric"  type="select" label="Dissimilarity metric" help="Metric to be used for calculating dissimilarities between observations, available 'euclidian' or 'manhattan'? " > 
+					<option value="euclidian" selected="true">euclidian</option>
+					<option value="manhatan">manhatan</option>
+				</param>
+			    <param name="HCPC_method"  type="select" label="Clustering method" help="Clustering method between 'ward', 'average', 'single', 'complete', 'weighted' " > 
+					<option value="ward" selected="true">ward</option>
+					<option value="average">average</option>
+					<option value="single">single</option>
+					<option value="complete">complete</option>
+					<option value="weighted">weighted</option>
+				</param>
             </when>
             <when value="PCA">
+            	  <param name="PCA_npc" value="5" type="integer" label="Number of principal components to keep"
+                       help="The number of dimensions which are kept for PCA analysis (default=5)" />
             </when>
+            
         </conditional>
+        	<section name="coordinates" type="section" label="Table with scatter plots coordinates" expanded=FALSE>
```

Comment:
The tag `section` has no attribute `type` and no attribute `label`, see the [doc](https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-inputs-section).

Better using `planemo lint .` to debug this type of issues

`<section name="coordinates" title="Table with scatter plots coordinates" expanded=FALSE>`

Reply:
OK j'ai encore fait des modifs
Le xml me parait OK
Les tests ne passent pas sur Ged-Serveur (il ne retrouve pas la librairie optparse... Je dois voir ce qui se passe plus en details 💃